assets/queries/cloudFormation/clientsecret_is_exposed/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "clientsecret_is_exposed",
+  "queryName": "ClientSecret is Exposed",
+  "severity": "MEDIUM",
+  "category": "Identity and Access Management",
+  "descriptionText": "ClientSecret is not a plaintext string, also does not start with '{{resolve:secretsmanager:' or '{{resolve:ssm-secure:'",
```

Comment:
`ClientSecret is not a plaintext string and it doesn't use 'AWS Systems Manager Parameter Store' nor 'AWS Secrets Manager'`